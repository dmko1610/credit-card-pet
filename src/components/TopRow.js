

const TopRow = () => {
  return (
    <View
      style={{
        flex: 1,
        flexDirection: "row",
        justifyContent: "space-between",
        margin: 10,
      }}
    >
      <Hologram />
      <PaySystemLogo />
    </View>
  );
};

export default TopRow;
