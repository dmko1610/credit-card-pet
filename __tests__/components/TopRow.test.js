import React from "react";
import renderer from "react-test-renderer";

import TopRow from "../../src/components/TopRow";

describe("Top Row Creation", () => {
  it("Render component", () => {
    const tree = renderer.create(<TopRow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
