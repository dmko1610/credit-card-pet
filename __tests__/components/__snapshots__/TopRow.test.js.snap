// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Row Creation Render component 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "margin": 10,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/hologram.png",
      }
    }
    style={
      Object {
        "height": 50,
        "width": 50,
      }
    }
  />
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/mastercard.png",
      }
    }
    style={
      Object {
        "height": 60,
        "width": 60,
      }
    }
  />
</View>
`;
